src/components/layout/WithVehicleReference.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { FC, PropsWithChildren } from 'react';
+
+import { Center, chakra, Grid, GridItem } from '@chakra-ui/react';
+
+import VehicleReference, { VehicleProps } from '../vehicleReference';
+import Stack from '../stack';
+import Link from '../link';
+import { ArrowLeftIcon } from '../icons';
+
+interface Props {
+  title: string;
+  backLink?: {
+    text: string;
+    url: string;
+  };
+  vehicle: VehicleProps;
+}
+
+const LayoutWithVehicleReference: FC<PropsWithChildren<Props>> = ({
+  title,
+  backLink,
+  vehicle,
+  children,
+}) => {
+  const Component = chakra('main');
+
+  return (
+    <Center>
+      <Component maxWidth="container.xl">
```

Comment:
Should we use the container component instead? https://chakra-ui.com/docs/components/container

Reply:
Also don't we need to ensure the container takes the full 1280px?

Reply:
Wouldn't `<Container as="main">` do the trick?